build/azDevOps/azure/ci.yml

```diff
@@ -131,7 +166,32 @@ stages:
       - name: core_environment
         value: nonprod
     jobs:
+      - job: canideploy_dev
+        condition: eq(variables.run_contract_test, true)
+        steps:
+          - task: Bash@3
+            displayName: 'Pull Pact CLI Docker image'
+            inputs:
+              targetType: inline
+              script: |
+                docker pull pactfoundation/pact-cli:latest
+
+          - task: Bash@3
+            displayName: 'Contract Tests: can-i-deploy to dev'
+            inputs:
+              targetType: inline
+              script: |
+                docker run --rm \
+                  -e PACT_BROKER_BASE_URL=$(PACT_BROKER_BASE_URL) \
+                  -e PACT_BROKER_TOKEN=$(PACT_BROKER_TOKEN) \
+                  pactfoundation/pact-cli:latest \
+                  broker can-i-deploy \
+                  --pacticipant $(PACTICIPANT_NAME) \
+                  --version $(version_number) \
+                  --to-environment $(Environment.ShortName)
+
       - deployment: AppInfraDev
+        dependsOn: canideploy_dev
```

Comment:
Is this needed? It's not in the the other pipeline files?

Reply:
If it is can you test that it still runs when the `run_contract_tests` flag is false?

Reply:
Ah, forgot about that, thanks. 

Will test it out, might need some custom condition here...

Reply:
I've changed this to a condition and added it on other deploy jobs.